runtime/oti/ObjectHash.hpp

```diff
@@ -203,14 +203,89 @@ class VM_ObjectHash
 		return inlineConvertValueToHash(vm, (UDATA)objectPointer);
 	}
 
+	/**
+	 * Fetch objectPointer's hashcode when objectPointer is a contiguous object.
+	 *
+	 * @pre objectPointer must be a valid object reference
+	 *
+	 * @param vm			a java VM
+	 * @param objectPointer 	a valid object reference
+	 * @param objectClass 	class of objectPointer
+	 * @param offset 	objectPointer size offset
+	 */
+	static VMINLINE I_32
+	inlineIndexableContiguousObjectHashCode(J9JavaVM *vm, j9object_t objectPointer, J9Class *objectClass, UDATA offset)
+	{
+		J9ROMArrayClass *romClass = (J9ROMArrayClass *)objectClass->romClass;
+		offset = ROUND_UP_TO_POWEROF2((offset << (romClass->arrayShape & 0x0000FFFF)) + vm->contiguousIndexableHeaderSize, sizeof(I_32));
```

Comment:
Isn't there a name for `0x0000FFFF` that should be used here?

Reply:
I wouldn't say "all over", but there are a few places that would benefit from a macro:
```c
#define J9ARRAYCLASS_LOG_ELEMENT_SIZE(clazz) \
	(((J9ROMArrayClass *)((clazz)->romClass))->arrayShape & 0x0000FFFF)
```
(based on [this line in ArrayCopyHelpers.hpp](https://github.com/eclipse-openj9/openj9/blob/master/runtime/oti/ArrayCopyHelpers.hpp#L584) and other similar occurrences).

Reply:
Okay, should this be added to this PR?